Find404.scala

```diff
@@ -0,0 +1,16 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+
+object Find404 {
+    
+    def main(args: Array[String]): Unit = {
+        println("Provide a directory")
+        val directory = new File(readLine())
+
+        val directorySet = directory.listFiles()
+        val fileSet = directorySet.filter(_.getClass().getName().indexOfSlice(".") == -1)
```

Comment:
This is overly complicated, it can just be:
```
val fileSet: Array[File] = directorySet.filter(file => file.isFile)
```

But this won't go into subsequent folders to find more files. And a simple loop won't go into the folder's folders to list more files. This is the sort of problem that lends itself nicely to recursion. See some approaches to this [here](https://stackoverflow.com/questions/2637643/how-do-i-list-all-files-in-a-subdirectory-in-scala)